indico/queries/jobs.py

```diff
@@ -67,11 +68,17 @@ class JobStatus(RequestChain):
 
     previous: Job = None
 
-    def __init__(self, id: str, wait: bool = True, request_interval=0.2, timeout=None):
+    def __init__(
+        self,
+        id: str,
+        wait: bool = True,
+        timeout: Tuple[int, float] = None,
+        max_wait_time: Tuple[int, float] = 5,
```

Comment:
what's the diff? (also Union)

Reply:
also - docstring update?

Reply:
timeout makes call throw `IndicoTimeoutError` after a specified period of time. `max_wait_time` denotes the max wait time per retry call until JobStatus is COMPLETE. 

Updated docstring to hopefully make the distinction clear

Reply:
`retry_time_interval` or `time_between_retries` would be a clearer name for this variable I think.

Reply:
changed `max_wait_time` to `request_interval` to match existing argument in `JobStatus` query